tsconfig.json

```diff
@@ -3,7 +3,6 @@
     "experimentalDecorators": true,
     "target": "esnext",
     "module": "esnext",
-    "strict": true,
```

Comment:
not sure if that should be deleted, sounds quite important for typechecking: https://www.typescriptlang.org/tsconfig#strict

Reply:
This will be gone with the upgrade to Vue3. There's a TS issue with how it handles how strict the code is in our tests that's solved by updating